src/main/java/ch/bfh/bti7081/s2018/green/models/entities/RecurringEvent.java

```diff
@@ -0,0 +1,30 @@
+package ch.bfh.bti7081.s2018.green.models.entities;
+
+import javax.persistence.DiscriminatorValue;
+import javax.persistence.Entity;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+import java.time.LocalDateTime;
+
+
+@Entity
+@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
```

Comment:
does this mean, that there is only one table for all events (normal and recurring)? is this why there is a field in the event class that is set to null by default?